specification/deviceregistry/DeviceRegistry.Management/examples/2025-10-01/Update_SchemaRegistry.json

```diff
@@ -43,7 +43,8 @@
     },
     "202": {
```

Comment:
question: why do we have a 202 for PATCH schema registry? i thought it was a "synchronous" solution and returns 200? shouldn't it match the CreateOrReplace example for schema registry?